client/transaction-pool/graph/src/pool.rs

```diff
@@ -84,6 +86,9 @@ pub trait ChainApi: Send + Sync {
 
 	/// Returns hash and encoding length of the extrinsic.
 	fn hash_and_length(&self, uxt: &ExtrinsicFor<Self>) -> (Self::Hash, usize);
+
+	/// Return body
```

Comment:
```suggestion
	/// Returns a block body given the block id.
```

To match other docs in the trait.